safety/cli.py

```
@@ -776,14 +778,25 @@ def check_updates(ctx: typer.Context,
+                )
+                console.print()
+                console.print(f"Pip: To install the updated version of Safety directly via pip, run: pip install safety=={latest_available_version}")
+            elif packaging_version.parse(latest_available_version) < packaging_version.parse(VERSION):
+                # Notify user about downgrading
+                console.print(f"Latest stable version is {latest_available_version}. If you want to downgrade to this version, you can run: pip install safety=={latest_available_version}")
+            else:
+                console.print("You are already using the latest stable version of Safety.")
+        except InvalidVersion as invalid_version:
+            LOG.exception(f'Invalid version format encountered: {invalid_version}')
+            console.print(f"Error: Invalid version format encountered for the latest available version: {latest_available_version}")
+            console.print(f"Please report this issue or try again later.")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)